source/getstarted.html.erb

```diff
@@ -0,0 +1,59 @@
+---
+title: Get started - GovWifi
+---
+<nav class="breadcrumbs" aria-label="Breadcrumbs" data-click-events data-click-category="Header" data-click-action="Breadcrumb clicked">
+  <ol itemscope itemtype="http://schema.org/BreadcrumbList">
+    <li class="breadcrumbs__item breadcrumbs__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+      <a href="/" itemprop="item"><span itemprop="name">GovWifi</span></a>
+    </li>
+    <li class="breadcrumbs__item breadcrumbs__item--active" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+      <a href="#main" itemprop="item"><span itemprop="name">Get started</span></a>
+    </li>
+  </ol>
+</nav>
+<div class="container">
+  <div class="grid-row">
+    <div class="column-two-thirds column-minimum">
+      <h1 class="heading-large" style="padding-bottom: 15px" >Get started</h1>
```

Comment:
Is this inline style to make it line up?
Nothing we can use with predefined styles?

Reply:
The predefined styles weren't applying. So I used inline.

Reply:
I was to give spacing between the different sections

Reply:
Does it look awful if we didn't apply the inline styles?
A bit worried that we are missing predefined styles that can do this for us.

Reply:
Ive used the predefined styles now. It does not look bad imo, was just being a perfectionist.

Reply:
This is using the predefined styles @emileswarts 

<img width="546" alt="screen shot 2018-10-09 at 12 20 55" src="https://user-images.githubusercontent.com/40758489/46666168-e077b880-cbbd-11e8-9c16-5824884772ce.png">